src/plugins/git/README.md

```diff
@@ -0,0 +1,16 @@
+# GIT plugin
+
+## Goal
+The cmd plugin offers user to download code from git repo.
+
+## Schema
+```yaml
+extras:
+  com.microsoft.pai.runtimeplugin:
+    - plugin: git
+      parameters:
+        repo_uri: <git repo>
+        options:
+        - <git clone options>
+        clone_dir: <clone dir>
```

Comment:
If the user doesn't give `clone_dir`, the plugin will use `/code` (or `~/code`? I'm not quite sure) as the default dir. Please document the behavior.

Reply:
Will use path `/usr/local/pai/code`, will doc this

Reply:
Added doc, thx.